packages/testsuite/cypress/support/form-editing.ts

```diff
@@ -295,20 +304,32 @@ declare global {
        * @param attributeName - specific ID part of form input with text form input.
        */
       clearAttribute(formId: string, attributeName: string): Chainable<void>;
+      /**
+       * Select value from <select> form input.
```

Comment:
We should document that the input is usually not part of the main edit form.

Reply:
Documentation updated.